examples/vue/nuxt-app/nuxt.config.js

```diff
@@ -0,0 +1,61 @@
+
+export default {
+  mode: 'spa',
+  /*
```

Comment:
Could I suggest that you either add `server: { port: 4000 }` here to match the tauri conf, or edit the tauri conf to ``"devPath": "http://localhost:3000"` (the default Nuxt dev port)?